src/main/webapp/WEB-INF/tags/instructor/feedbackEdit/questionFeedbackPathSettings.tag

```diff
@@ -5,74 +5,113 @@
 <%@ tag import="teammates.common.datatransfer.FeedbackParticipantType" %>
 
 <%@ attribute name="fqForm" type="teammates.ui.template.FeedbackQuestionEditForm" required="true"%>
+<c:set var="isNewQuestion" value="${empty fqForm.questionNumberSuffix}" />
 
 <div class="col-sm-12 padding-15px margin-bottom-15px background-color-light-green">
     <div class="col-sm-12 padding-0 margin-bottom-7px">
         <b class="feedback-path-title">Feedback Path</b> (Who is giving feedback about whom?)
     </div>
-    <div class="col-sm-12 col-lg-6 padding-0 margin-bottom-7px"
-        data-toggle="tooltip" data-placement="top"
-        title="<%= Const.Tooltips.FEEDBACK_SESSION_GIVER %>">  
-        <label class="col-sm-4 col-lg-5 control-label">
-            Who will give the feedback:
-        </label>
-        <div class="col-sm-8 col-lg-7">
-            <select class="form-control participantSelect"
-                id="<%= Const.ParamsNames.FEEDBACK_QUESTION_GIVERTYPE %>${fqForm.questionNumberSuffix}"
-                name="<%= Const.ParamsNames.FEEDBACK_QUESTION_GIVERTYPE %>"
-                <c:if test="${!fqForm.editable}">disabled</c:if>
-                onchange="matchVisibilityOptionToFeedbackPath(this)">
-                <c:forEach items="${fqForm.feedbackPathSettings.giverParticipantOptions}" var="option">
-                    <option ${option.attributesToString}>
-                        ${option.content}
-                    </option>
-                </c:forEach>
-            </select>
-        </div>
+    <div class="feedback-path-dropdown btn-group">
+        <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <c:choose>
+                <c:when test="${isNewQuestion}">
+                    Please select a feedback path <span class="caret"></span>
+                </c:when>
+                <c:when test="${fqForm.feedbackPathSettings.commonPathSelected}">
+                    ${fqForm.feedbackPathSettings.selectedGiver.displayNameGiver} will give feedback on <span class='glyphicon glyphicon-arrow-right'></span> ${fqForm.feedbackPathSettings.selectedRecipient.displayNameRecipient}</span>
+                </c:when>
+                <c:otherwise>
+                    Predefined combinations:
+                </c:otherwise>
+            </c:choose>
+        </button>
+        <ul class="dropdown-menu">
+            <li class="dropdown-header">Common feedback path combinations</li>
+            <c:forEach items="<%= Const.FeedbackQuestion.COMMON_FEEDBACK_PATHS %>" var="commonPath">
+                <li class="dropdown-submenu">
+                    <c:set var="commonGiver" value="${commonPath.key}" />
+                    <a>${commonGiver.displayNameGiver} will give feedback on...</a>
```

Comment:
I think actually since the recipient may not be able to see the response, we should use `on` instead of `to`